components/UnicornPicture.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+<div class="picture-unicorn">
+    <img v-if="found" class="picture-unicorn__photo" :src="srcImage">
+    <img v-else class="picture-unicorn__photo" src="~assets/icons/not-found-image.jpg">
+</div>
+</template>
+<script>
+    export default{
+        name: "UnicornPicture",
+        props:{
+            unicorn: {
+                type: Object,
+                required: true
+            }
+        },
+        computed:{
+            found(){
+                return this.unicorn.photo != null;
+            },
+
+            srcImage(){
```

Comment:
Du genre
if (this.unicorn.photo) {
 return  '~assets/icons/not-found-image.jpg'
}
return process.env.baseUrl + this.unicorn.photo.url;